src/Examples/BigEndianReader.cs

```diff
@@ -0,0 +1,43 @@
+using System;
```

Comment:
Tbh we should likely move all typical datasets into a new package TorchSharp.Data, much as we do in DiffSharp

Can be done in separate PR

Would also like to see most samples done using notebooks, which are pretty great to use in VS Code

Reply:
Setting up a separate TorchSharp.Data package seems like a good idea.